### DISCUSSION TITLE
Binding clicks on the group bar?

### DISCUSSION THREAD

**Discussion by djarb** [hyprwm/Hyprland#4530] [Q&A] [answered]
It seems like there's a default binding for middle clicks on the group bar to close the window. I'd like to change that, but I'm not finding any way to do it in the wiki. Any clues would be welcome.

Bonus points if you can tell me how to move the group bar to the bottom of the group window instead of the top :)

**Comment by MightyPlaza** [comment, marked as answer]
> It seems like there's a default binding for middle clicks on the group bar to close the window. I'd like to change that, but I'm not finding any way to do it in the wiki. Any clues would be welcome.

not possible https://github.com/hyprwm/Hyprland/pull/4297
 
> Bonus points if you can tell me how to move the group bar to the bottom of the group window instead of the top :)

not yet possible https://github.com/hyprwm/Hyprland/pull/3197
